index.js

```diff
@@ -270,6 +276,8 @@ Parser.parseString = function(xml, options, callback) {
       return parseRSS2(result, options, callback);
     } else if (result['rdf:RDF']) {
       return parseRSS1(result, options, callback);
+    } else if (result.rss && result.rss.$.version && result.rss.$.version.indexOf('9') === 2) {
```

Comment:
Just to make this a little more robust, can you change `.indexOf('9') === 2` to `.match(/0\.9/)`?

Reply:
sure